### ISSUE TITLE
Windows 11 download does not work

### ISSUE THREAD

**Issue by kekonn** [wimpysworld/quickemu#240]
#### Expected behaviour

When I run `quickget windows 11` I expect the script to set up a Windows 11 vm

#### Actual behaviour

It produces the following output:

```
╭─kekkon@tuxgamer in ~/Documents/vms took 1s
[🔴] × quickget windows 11
Getting Windows 11 URL...
SSL_INIT
SSL_INIT
SSL_INIT
SSL_INIT
SSL_INIT
Downloading Windows 11 [22000.318]...
SSL_INIT
SSL_INIT
ERROR! Failed to download https://tb.rg-adguard.net/dl.php?go=55a5f243. Try running 'quickget' again.
```

Extra diagnostics:
```
╭─kekkon@tuxgamer in ~/Documents/vms took 4ms
╰─λ dig @9.9.9.9 tb.rg-adguard.net

; <<>> DiG 9.16.23 <<>> @9.9.9.9 tb.rg-adguard.net
; (1 server found)
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 34225
;; flags: qr rd ra; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 1232
;; QUESTION SECTION:
;tb.rg-adguard.net.             IN      A

;; ANSWER SECTION:
tb.rg-adguard.net.      300     IN      A       163.172.251.201

;; Query time: 24 msec
;; SERVER: 9.9.9.9#53(9.9.9.9)
;; WHEN: Wed Nov 17 17:59:29 CET 2021
;; MSG SIZE  rcvd: 62
```

#### Steps to reproduce the behaviour
Run `quickget windows 11`

#### Quickemu output

Run `quickemu` or `quickget` and paste the output here.

See Actual Behaviour

#### Linux Distribution & Kernel

Run the following and paste the output.

```bash
lsb_release --all
uname -a
```

```
╭─kekkon@tuxgamer in ~/Documents/vms took 295ms
╰─λ lsb_release --all
LSB Version:    1.4
Distributor ID: Garuda
Description:    Garuda Linux
Release:        Soaring
Codename:       Harpy-Eagle

╭─kekkon@tuxgamer in ~/Documents/vms took 4ms
╰─λ uname -a
Linux tuxgamer 5.15.2-zen1-1-zen #1 ZEN SMP PREEMPT Fri, 12 Nov 2021 19:22:07 +0000 x86_64 GNU/Linux
```

**Comment by philclifford** [comment]
Thanks : see #239 which will be 
fixed by #241

**Comment by flexiondotorg** [comment]
Fixed via #241
